plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3310,4 +3319,4 @@ notification.update_available.msg=There is a new version of MCreator available.<
 notification.patch_available.msg=There is a more recent version of {0} available.<br>\
   <br>Installed version: {0}.{1}\
   <br>New version: {0}.{2}
-notification.plugin_updates.msg=There are some plugin updates available
\ No newline at end of file
+notification.plugin_updates.msg=There are some plugin updates available
```

Comment:
This is an unnecessary change.

Reply:
I didn't change that translation line, I just added the translations of the new procedures, what do you want me to do, to that translation line?

Reply:
According to GitHub, there's a change. Did you remove the line after this one? If yes, then that's the issue.

Reply:
I haven't changed it, but I think github does because I added some new lines of translation and that is no longer at the same line (it's happened to me also in my other generators)

Reply:
As I said that's not the issue. It's about this line being the last line of the file. You deleted the empty line that was there before, hence the change.

Reply:
How can I solve it?